lib/cosmos/tools/tlm_viewer/tlm_viewer.rb

```diff
@@ -433,9 +433,8 @@ def closeEvent(event)
           event.ignore()
         else
           # Close any open screens
-          script_disconnect()
```

Comment:
data_viewer.rb also calls script_disconnect. Does it need to change? Should script_disconnect go away completely?